Source/JavaScriptCore/runtime/IntlDurationFormat.cpp

```diff
@@ -406,6 +426,18 @@ static Vector<Element> collectElements(JSGlobalObject* globalObject, const IntlD
                 return String(WTFMove(buffer));
             };
 
+            // https://github.com/unicode-org/icu/blob/main/docs/userguide/format_parse/numbers/skeletons.md#sign-display
+            if (!signDisplayed)
+                skeletonBuilder.append(" +_"_s);
+            else if (!value) {
+                if (!durationSign)
+                    durationSign = getDurationSign(duration);
+                if (durationSign == DurationSignType::Negative) {
```

Comment:
Interesting; I would have thought that this would be constant `false` because the LHS and RHS types are different, but apparently it works. I still feel like it would be clearer to write `durationSign.value()`, but I don't think we have a style rule about this, so maybe it's okay.